lib/surface/catalogue/components/playground_tools.ex

```diff
@@ -61,6 +62,15 @@ defmodule Surface.Catalogue.Components.PlaygroundTools do
             </Form>
           </div>
         </TabItem>
+        <TabItem label="Slots" visible={@slots != []}>
```

Comment:
maybe instead of using `visible` to show or not this `TabItem` should be better to use the `:if={}` statement and reduce the amount of coding to be rendering, WDTY?